packages/create-remix/__tests__/create-remix-test.ts

```diff
@@ -882,6 +862,282 @@ describe("create-remix CLI", () => {
     process.env.npm_config_user_agent = originalUserAgent;
   });
 
+  it("works when creating an app in the current dir", async () => {
+    let emptyDir = getProjectDir("current-dir-if-empty");
+    fse.mkdirSync(emptyDir);
+
+    let { status, stderr } = await execCreateRemix({
+      cwd: emptyDir,
+      args: [
+        ".",
+        "--template",
+        path.join(__dirname, "fixtures", "stack"),
+        "--no-git-init",
+        "--no-install",
+      ],
+    });
+
+    expect(stderr.trim()).toBeFalsy();
+    expect(status).toBe(0);
+    expect(fse.existsSync(path.join(emptyDir, "package.json"))).toBeTruthy();
+    expect(fse.existsSync(path.join(emptyDir, "app/root.tsx"))).toBeTruthy();
+  });
+
+  it("does not copy .git nor node_modules directories if they exist in the template", async () => {
```

Comment:
@markdalgleish I decided to treat both `.git` and `node_modules` the same and not copy either of them from the template directory.  We probably shouldn't promote bloated templates with committed `node_modules` that we have to download.